fix: adding the ability to set the ingress tlsSecret (#41426)
keycloak/keycloak: docs/documentation/release_notes/topics/26_4_0.adoc

This helps you to track down a warning or error message in the log to a specific caller or environment

For more details on this opt-in feature, see the https://www.keycloak.org/server/logging[Logging guide].

> = Ability to specify a tlsSecret on the Keycloak CR ingress

Reviewer: ```suggestion
= Ability to specify a `tlsSecret` for Ingress in Keycloak CR
```

IMHO this might be better. WDYT?

Reviewer: That's right - agree.

> In order to support basic TLS termination (edge) deployments via the operator, you may now set the Keycloak CR `spec.ingress.tlsSecret` field to a tls Secret name in the namespace.

Reviewer: ```suggestion
In order to support basic TLS termination (edge) deployments via the operator, you may now set the Keycloak CR `spec.ingress.tlsSecret` field to a TLS Secret name in the namespace.
```
